src/azul/__init__.py

```diff
@@ -820,6 +820,10 @@ def reindex_sources(self) -> List[str]:
 
     terms_aggregation_size = 99999
 
+    @property
+    def google_oauth_client_id(self) -> str:
+        return os.environ['AZUL_GOOGLE_OAUTH_CLIENT_ID']
```

Comment:
```suggestion
        return os.environ.get('AZUL_GOOGLE_OAUTH_CLIENT_ID')
```
Is it a requirement that all deployments have this property set?

Reply:
It's going to complicate the Swagger configuration if it's allowed to be unset, so I'll say "yes" until instructed otherwise.